docs/pages/translations.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module Pages
+	class Translations < ApplicationPage
```

Comment:
I added a new page instead an small section, because Translation may end up having more features (eg. Generating `.yml` files).

English is not my primary language, so there must be errors in the text, and there must also be better ways of writing, so can you help me and suggest writings, I will be grateful.